test/requirements/python-requirements.spec.js

```diff
@@ -0,0 +1,28 @@
+/*jshint node:true, mocha:true*/
+
+'use strict';
+const { spawnSync } = require('child_process');
+const assert = require('assert');
+const fs = require('fs');
+const path = require('path');
+
+describe('python-test', function () {
+    it('Should find python 3', (done) => {
+        const processOutput = spawnSync(`python3`, ['-c', `"print('hello world')";`]);
+        assert(processOutput.status === 0);
+        done();
+    });
+
+    describe('requirements-test', function() {
+        let requirements = fs.readFileSync(path.join(__dirname, '..', '..', 'requirements.txt'),
+            {encoding: 'utf-8'}).split('\n').filter((val) => val);
```

Comment:
encoding can be passed as the second argument to `readFileSync` and `.trim()` might be more concise if you are only worried about leading/trailing new lines. (If you are worried about intermediate empty lines, `.map(line => line.trim()).filter(line => !!line)` might be better.)